app/services/account_search_service.rb

```diff
@@ -92,6 +90,8 @@ def from_elasticsearch
     ActiveRecord::Associations::Preloader.new.preload(records, :account_stat)
 
     records
+  rescue Faraday::ConnectionFailed, Parslet::ParseFailed
```

Comment:
I agree with this, but not the other change, because for 0 results it will double the work load

Reply:
Then, how about return nil to prevent double work?
ES connection failed → return nil → do Database search
ES returned 0 results → Just use that

Reply:
Or just return empty result without fallback
How is your opinion?
I like to use first one to make giving results as possible